src/detail-panels/lists/list-view.jsx

```diff
@@ -43,14 +60,42 @@ function ListViewEntry({ className, entry }) {
           noTooltip
           className='list-add-date' />
       </div>
-      <div className='row'>
+      {/* <div className='row'  > */}
+      <div>
         <span className='list-name'>
-          <a href={entry.url} target='__blank'>
+          <a href={entry.url} target='__blank' style={{opacity}}>
           {entry.name}
           </a>
+          {entry.spam && (
+            <ClickAwayListener onClickAway={handleTooltipClose}>
+              <Tooltip 
+                title={`Flagged as spam. Source: ${entry.source || 'unknown'}`}
+                onClose={handleTooltipClose}
+                open={open}
+                disableFocusListener
+                disableHoverListener
+                disableTouchListener
+                slotProps={{
+                  popper: {
+                    disablePortal: true,
+                    sx: {
+                      [`&.${tooltipClasses.popper}[data-popper-placement*="bottom"] .${tooltipClasses.tooltip}`]:
+                        {
+                          marginTop: '0px',
+                        },
+                    },
+                  },
+                }}
```

Comment:
there has to be a simpler way to do this positioning override, but we can figure that out later